### ISSUE TITLE
Discriminator

### ISSUE THREAD

**Issue by ydnaandy123** [xhujoy/CycleGAN-tensorflow#18]
Hi very nice work and thanks for sharing!
I noticed that there is no any normalization after the first convolution of Discriminator.
In my opinion this won't matter a lot. 
But are there any theoretically reasons or empirical  results making you choose so?
I'll be glad if you reply me~
And thank you again!

**Comment by xhujoy** [comment]
In my opinion, this usage is firstly mentioned in [DCGAN](https://arxiv.org/pdf/1511.06434.pdf). `Directly applying batchnorm to all layers however, resulted in sample oscillation and model instability. This was avoided by not applying batchnorm to the generator output layer and the discriminator input layer.`

**Comment by ydnaandy123** [comment]
Oh wow!
Didn't notice that before~
Thank you!!
